tests/Unit/NumericalAlgorithms/Interpolation/Test_ObserveSurfaceIntegrals.cpp

```diff
@@ -113,31 +131,40 @@ struct MockInterpolator {
 };
 
 struct MockMetavariables {
-  struct InterpolationTargetA {
+  struct SurfaceA {
     using compute_items_on_source = tmpl::list<>;
     using vars_to_interpolate_to_target =
         tmpl::list<Tags::TestSolution,
                    gr::Tags::SpatialMetric<3, Frame::Inertial>>;
     using compute_items_on_target = tmpl::list<Tags::SquareComputeItem>;
     using compute_target_points =
-        intrp::Actions::KerrHorizon<InterpolationTargetA, ::Frame::Inertial>;
+        intrp::Actions::KerrHorizon<SurfaceA, ::Frame::Inertial>;
```

Comment:
the fact that this is a horizon interpolator, but the integration seems to be performed on flat space is a bit confusing. Perhaps a comment where the Minkowski metric is invoked?